DigitalLearningSolutions.Web/webpack.config.js

```diff
@@ -53,7 +56,21 @@ const config = {
     ],
   },
   plugins: [
-    new ForkTsCheckerWebpackPlugin(),
+    new ForkTsCheckerWebpackPlugin({

```

Comment:
Taken from the [babel-loader](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/tree/main/examples/babel-loader) example, since that is the loader we are using.

I think I added the extra config (mode, issue) to fix some problems that were occurring during piecemeal dep updates, but these can be removed.